apps/src/Sounds.js

```diff
@@ -344,6 +352,7 @@ Sounds.prototype.playBytes = function(bytes, playbackOptions) {
   soundConfig.playAfterLoadOptions = playbackOptions;
   soundConfig.bytes = bytes;
   let sound = new Sound(soundConfig, this.audioContext);
+  this.soundsById[id] = sound;
```

Comment:
Fixes an unrelated bug where clicking "reset" didn't stop TTS sounds because they weren't tracked in `this.soundsById`.